.github/workflows/ci.yml

```diff
@@ -195,3 +200,20 @@ jobs:
           publish_branch: gh-pages
           publish_dir: ./${{ matrix.dist_path }}
           force_orphan: true
+
+      - name: Prepare Release (Tauri)
+        if: matrix.platform == 'tauri'
+        shell: bash
+        run: .github/workflows/build_zip.sh
+        env:
+          OS_NAME: ${{ matrix.os }}
+          TARGET: ${{ matrix.binaryen }}
+
+      - name: Release (Tauri)
+        if: matrix.platform == 'tauri'
+        uses: softprops/action-gh-release@v2
+        with:
+          files: LiveSplitOne-*.*
+          name: Latest
+          tag_name: latest
+          body: This contains the latest version of the auto splitter.
```

Comment:
Is this supposed to say "auto splitter"?

Reply:
Oh, no, I just copied it from our auto splitter template. But as with all things CI, this will fail a couple of times anyway, so I didn't pay too much attention to minor stuff like this yet, but yeah this needs to be fixed.